src/compiler/crystal/compiler.cr

```diff
@@ -547,6 +559,16 @@ module Crystal
           puts " - #{unit.original_name} (#{unit.name}.bc)"
         end
       end
+
+      if units.size != reused.size
+        puts
+        puts("Top 10 slowest modules:")
+        units.sort_by! { |u| u.compilation_time * -1 }
+        units.first(10).each do |unit|
+          time_str = unit.compilation_time.to_s.ljust(18, ' ')
+          puts " - #{unit.compilation_time} #{unit.original_name} (#{unit.name}.bc)"
```

Comment:
Perhaps slightly more readable:

```suggestion
          puts " - %s %s (%s.bc)" % {unit.compilation_time, unit.original_name, unit.name}
```

Reply:
It interpolates the string using `sprintf` under-the-hood, see [`String#%`](https://crystal-lang.org/api/1.7.0/String.html#%(other):String-instance-method) :)

Reply:
It most likely will be less efficient tho', `sprintf` + tuple vs single string interpolation... :(

Reply:
I agree, KISS :)

Reply:
I'd like to point out that performance *really* should not matter for this. This is a single output line. Whether that runs 5ns faster or not makes no difference at all.